lib/class-experimental-wp-widget-blocks-manager.php

```diff
@@ -271,4 +292,93 @@ public static function serialize_block( $block ) {
 			);
 		}
 	}
+
+	/**
+	 * Outputs a block widget on the website frontend.
+	 *
+	 * @param array $options   Widget options.
+	 * @param array $arguments Arguments array.
+	 */
+	public static function gutenberg_output_blocks_widget( $options, $arguments ) {
```

Comment:
Now that these methods are inside the experimental class, they don't need the `gutenberg_` prefix.

Reply:
Changed in 300e1c250.

Reply:
Renamed in 300e1c250.